src/main/java/frc/robot/Robot.java

```diff
@@ -74,6 +84,9 @@ public void robotInit() {
 
     //DRIVE_TRAIN_SUBSYSTEM.setDefaultCommand(new DrivetrainDefaultCommand(DRIVE_TRAIN_SUBSYSTEM, GAMEPAD));
     SHOOTER_SUBSYSTEM.setDefaultCommand(new RunCommand(() -> SHOOTER_SUBSYSTEM.defaultCommand(), SHOOTER_SUBSYSTEM));
+    TURRET_SWIVEL_SUBSYSTEM.setDefaultCommand(TURRET_AIM_AT_TARGET);
+    TURRET_SWIVEL_SUBSYSTEM.setEncoder(0);
```

Comment:
Maybe this should be part of the subsystem constructor? Does Robot.java need to be aware of this initialization detail?

Reply:
Moved to subsystem constructor